aes-kw/src/kek.rs

```diff
@@ -117,6 +132,54 @@ where
         Kek { cipher }
     }
 
+    /// Very similar to the W(S) function defined by NIST in SP 800-38F,
+    /// Section 6.1
+    fn w(&self, n: usize, block: &mut GenericArray<u8, Aes::BlockSize>, out: &mut [u8]) {
+        for j in 0..=5 {
+            for (i, chunk) in out[IV_LEN..].chunks_mut(8).enumerate() {
```

Comment:
This is kind of six of one, half dozen of the other but...

```suggestion
            for (i, chunk) in out.chunks_mut(8).skip(1).enumerate() {
```

Reply:
Right, this only works if `IV_LEN` is equal to 8 (or `SEMIBLOCK_SIZE` as suggested by @SeanWykes). Which, we know to be true of course, and it probably won't change.

Reply:
Re-reading the RFC, I agree that `out.chunks_mut(SEMIBLOCK_SIZE).skip(1).enumerate()` better conveys the intent of `For i = 1 to n-1`.